hubble-dist/assembly/travis/run-api-test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -ev
+
+TRAVIS_DIR=`dirname $0`
+
+behave hubble-be/src/test/python
```

Comment:
$TRAVIS_DIR/hubble-be/src/test/python ?

Reply:
No, just hubble-be/src/test/python, $TRAVIS_DIR is hubble-dist/assembly/travis/xxx